launch/launcher_windows.go

```diff
@@ -3,28 +3,13 @@ package launch
 import (
 	"os"
 	"os/exec"
-	"path/filepath"
-	"strings"
-	"syscall"
 )
 
 func OSExecFunc(argv0 string, argv []string, envv []string) error {
-	c := OSExecCmd(argv, envv)
+	c := exec.Command(argv[0], argv[1:]...)
+	c.Env = envv
 	c.Stdin = os.Stdin
 	c.Stdout = os.Stdout
 	c.Stderr = os.Stderr
 	return c.Run()
 }
-
-func OSExecCmd(argv []string, envv []string) *exec.Cmd {
-	var c *exec.Cmd
-	arg0 := filepath.Base(argv[0])
-	if arg0 == "cmd" || arg0 == "cmd.exe" {
-		c = exec.Command(argv[0])
```

Comment:
@ekcasey Are you sure it's safe to remove this branching? I forget the details but using `SysProcAttr` was necessary for proper parsing in some situations. Are you sure this change isn't related to the test breakage you noted on Windows?

Reply:
Working backwards from the code as I found it: I believe problem this branching was motivated by the fact that profile script invocations were being added as a single argument: e.g "call profile.bat" instead of "call" "profile.bat" and therefore quotes needed to be stripped and the default argument escaping in the go stdlib had to be circumvented to undo the incorrect tokenization of the profile invocations. However, that workaround removed _all_ tokenization.

If we do nothing, go will escape arguments correctly for us (adding quotes only when an arg contains a " " or tab charcter). We just need to ensure that the arguments we pass in are properly tokenized.

Reply:
https://github.com/buildpacks/lifecycle/blob/main/launch/launcher.go#L136

The linux implementation does something similar but the linux implementation is making a bash script which makes it slightly different from the windows shell logic (`cmd /c` can't take accept a batch script as a string afaik)